src/core/component.ts

```diff
@@ -113,17 +113,23 @@ export default class Component<P> extends React.PureComponent<P, any> {
     const options: Record<string, any> = {};
     const templates: Record<string, any> = {};
 
+    const knownTemplates: Record<string, any> = {};
+
+    const isReactProp = (propName: string) => propName === "children";
```

Comment:
I would rather name it more abstractly (`isIgnoredProp` or like) and use `object`\`array` instead:
```ts
const ignoredProps = {
  children = true
}
/* * * */
} else if (!ignoredProps [key]) {
```